pandas/io/tests/test_stata.py

```diff
@@ -1113,6 +1113,58 @@ def test_read_chunks_columns(self):
                 tm.assert_frame_equal(from_frame, chunk, check_dtype=False)
                 pos += chunksize
 
+    # GH 13631, add support for writing variable labels
+    def test_write_variable_labels(self):
+        original = pd.DataFrame({'a': [1, 2, 3, 4],
+                                 'b': [1.0, 3.0, 27.0, 81.0],
+                                 'c': ['Atlanta', 'Birmingham',
+                                       'Cincinnati', 'Detroit']})
+        original.index.name = 'index'
+        variable_labels = {'a': 'City Rank', 'b': 'City Exponent', 'c': 'City'}
+        with tm.ensure_clean() as path:
```

Comment:
you need to close the StataReader (I think we have an ensure_ for that somewhere)